src/tsolvers/lasolver/LASolver.h

```diff
@@ -151,8 +152,8 @@ class LASolver: public TSolver
     LVRef getVarForTerm(PTRef ref) const  { return laVarMapper.getVarByPTId(logic.getPterm(ref).getId()); }
     void notifyVar(LVRef);                             // Notify the solver of the existence of the var. This is so that LIA can add it to integer vars list.
 
-    // Most-infeasible branching heuristic
-    LVRef splitOnMostInfeasible(vec<LVRef> const &) const;
+    // Random splitting heuristic
+    LVRef splitOnRandom(vec<LVRef> const &);
```

Comment:
```suggestion
    LVRef splitOnRandom(vec<LVRef> const &) const;
```
This can be `const`, right?

Reply:
It changes `LASolver::seed`.